### PR TITLE
#160719357 Fix email verification bug

### PR DESCRIPTION
**Pull request by emmaadesile**
### What does this PR do?
Fix bug on email verification controller

### Description of the task to be completed
- Change the argument used to call sendVerificationEmail controller from `user` to `createdUser`
  `sendVerificationEmail(createdUser)`

### How can this be manually tested
- After cloning the app
- download and install postgres
- create a `.env` file using `envsample` file
- run `npm run migrate` to create migrate db models
- run `npm run start:dev`
- open postman and make a post request to `localhost:3000/api/users`
- fill the new user details `firstName, lastName, username, email, password`
- open your email to see the verification email
- click the link provided and your new account will be verified

### What are the relevant pivotal tracker stories?
[160719357](https://www.pivotaltracker.com/story/show/160719357)

### CODE REVIEW

**Review by gonuoha — APPROVED** [review]
LGTM

**Review by iamuchejude — APPROVED** [review]
Good work

File: server/controllers/emailVerificationController.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ dotenv.config();
 
 const secret = process.env.JWT_KEY;
 const baseURL = process.env.BASE_URL;
-const mailKey = process.env.MAIL_API_KEY;
+const mailKey = process.env.SENDGRID_API_KEY;
```
Comment (waleCloud) [inline server/controllers/emailVerificationController.js:12]: Hi, @emmaadesile can you please explain why this was needed.

**Review by d-beloved — APPROVED** [review]
LGTM

File: server/controllers/emailVerificationController.js (hunk 1 above)
Comment (emmaadesile) [reply server/controllers/emailVerificationController.js:12]: @waleCloud  The `mailKey` is SendGrid API KEY. It is needed by sendgrid to send the email

**Pull request merged**